projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -60,6 +60,88 @@ function register_block() {
 					),
 					'align'   => array( 'wide', 'full' ),
 				),
+				'attributes'      => array(
```

Comment:
I think @Automattic/jetpack-vulcan team just moved these attributes out to blocks.json as part of some refactoring of blocks — maybe that also busted translations?

Reply:
I think yes, in the json file we can't translate.

Reply:
Looks to affect few other blocks too: pf5801-Aw-p2/#comment-336 — @monsieur-z @jeherve fine to move attributes back to PHP while the localization issues are sorted out with json approach?

I'm not sure if this would affect anything else given architecture changes to blocks.